draft-ietf-scitt-architecture.md

```diff
@@ -627,43 +625,15 @@ RegistrationInfo = {
 ## Signed Statement Issuance
 
 There are many types of Statements (such as SBOMs, malware scans, audit reports, policy definitions) that Issuers may want to turn into Signed Statements.
-An Issuer must first decide on a suitable format to serialize the Statement payload, such as:
-
-- JSON-SPDX
-- CBOR-SPDX
-- SWID
-- CoSWID
-- CycloneDX
-- in-toto
-- SLSA
+An Issuer must first decide on a suitable format to serialize a Statement payload. For a software supply chain, payloads describing the software artifacts may,
+for example, include JSON-SPDX, CBOR-SPDX, SWID, CoSWID, CycloneDX, in-toto, SLSA or others.
```

Comment:
Perhaps we can retain the list and add a last line to say, other formats can be added in future

Reply:
It makes the document shorter. 

If you think these payloads are import to the draft it would be important to add references and to expand the acronyms.

Reply:
I agree we should give due weightage to these acronyms as the they are the building blocks using SCITT and essential elements of Supply Chain Security!

I would check with other reviewers what they think about this...

Reply:
That's fine for me. Use the list style,  add references and explain the acronyms.

Reply:
Please phrase as a dedicated issue and push a PR.
This is postponed after submission moratorium of IETF 117.